app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
Storing `self` in the `Person.people` dictionary assumes that each person has a unique name. If names are not unique, this will cause issues as the last person with the same name will overwrite the previous one.